packages/reports/addon/components/report-actions/export.ts

```diff
@@ -40,10 +41,11 @@ export default class ReportActionExport extends Component<Args> {
   }
 
   /**
-   * filename for the downloaded file
+   * filename along with the format for the downloaded file
    */
   get filename() {
-    return this.args.model.title;
+    const dateString = moment().format('YYYYMMDDTHHmmss');
+    return `${this.args.model.title}-${dateString}`;
```

Comment:
It looks like this code is already present

https://github.com/yavin-dev/framework/blob/master/packages/reports/addon/components/report-actions/export.ts#L48